### ISSUE TITLE
FadeIn issue on Android

### ISSUE THREAD

**Issue by jonathanroze** [oblador/react-native-animatable#401]
Hi,

I'm using latest version of React-native and react-native-animatable.

When I want to trigger animation like ref.current.fadeOut()

I have this error on Android only:

`Error while updating property 'backfaceVisibility' of a view managed by: RCTView`

<img width="497" alt="Capture d’écran 2023-04-04 à 11 23 43" src="https://user-images.githubusercontent.com/2753799/229748094-11cb31ae-77c0-4304-ac9d-30e0be54e2a4.png">

**Comment by elencho** [comment]
yes, this library is not newArch ready yet :/ I have same problem

**Comment by jonathanroze** [comment]
@elencho Did you use flipInX or flipInY method?

**Comment by elencho** [comment]
No, My issue is in react-native-modal, It doesn't work anymore

**Comment by jonathanroze** [comment]
I solved my issue by editing animation flipInX and flipInY and removing "backfaceVisibility" property!

Maybe react-native-modal use this property.

**Comment by yeakbaba** [comment]
In my case, the issue was the animation prop of the component and the imperative animation function I called were not in the same animation family. 

In other words, while my component is using flipInX as animation prop, `<Animatable.View animation="flipInX" ref={animRef}>`
when I call `tada()` as imperative function, `animRef.current?.tada()`. I got the same error.

Replacing flipInX with any attention-seekers (rubberBand, wooble, swing, bounce...), or tada with any flippers was my workaround.
